src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1827,6 +1827,12 @@ inline std::ostream& operator<<(std::ostream& stream, MessageRate val) {
  * also restore the default `*_OUTPUT_DIAGNOSTICS_MESSAGES` configuration option
  * value for that interface. See @ref ConfigType.
  *
+ * @note
+ * When specifying @ref ProtocolType::ALL, message ID @ref ALL_MESSAGES_ID must
+ * also be specified. Further, the rate must be set to either
+ * @ref MessageRate::OFF or @ref MessageRate::DEFAULT; this prevents the serial
+ * port from being overloaded.
```

Comment:
Remove the last part. That's true for embedded platforms, but this message isn't specifically for serial ports. All that's important here is noting what the behavior is.